drivers/at/at.c

```diff
@@ -141,11 +141,15 @@ ssize_t at_send_cmd_get_lines(at_dev_t *dev, const char *command,
         }
         else if (res > 0) {
             bytes_left -= res;
-            if ((res == (2 + keep_eol)) && (strncmp(pos, "OK", 2) == 0)) {
+            uint8_t len_ok = strlen(AT_RECV_OK);
+            uint8_t len_error = strlen(AT_RECV_ERROR);
```

Comment:
I think sizet_t iso uint8_t makes more sense

Reply:
I'm also wondering if `sizeof() - 1` wouldn't be better (compile time).

And below in the `strncmp()`, shouldn't you check if `len_ok` / `len_error` is `!= 0` ?